config/environments/production.rb

```diff
@@ -16,6 +16,8 @@
   # Compress JavaScripts and CSS
   config.assets.compress = true
   config.assets.js_compressor = Uglifier.new(harmony: true)
+  Uglifier.new(harmony: true).compile(File.read("public/lib/leaflet-environmental-layers/dist/LeafletEnvironmentalLayers.js"))
+  Uglifier.new(harmony: true).compile(File.read("public/lib/leaflet-environmental-layers/lib/glify.js"))
```

Comment:
Will these then be grabbed automatically by asset pipeline? Awesome, thank you @Tlazypanda you're really an expert in all this!!!